src/libraries/System.Text.Json/gen/Resources/Strings.resx

```diff
@@ -171,4 +171,10 @@
   <data name="FastPathPolymorphismNotSupportedMessageFormat" xml:space="preserve">
     <value>Type '{0}' is annotated with 'JsonDerivedTypeAttribute' which is not supported in 'JsonSourceGenerationMode.Serialization'.</value>
   </data>
+  <data name="JsonStringEnumConverterNotSupportedTitle" xml:space="preserve">
+    <value>The non-generic 'JsonStringEnumConverter' requires dynamic code.</value>
+  </data>
+  <data name="JsonStringEnumConverterNotSupportedMessageFormat" xml:space="preserve">
+    <value>The member '{0}' has been annotated with 'JsonStringEnumConverter' which is not supported in Native AOT. Consider using the generic 'JsonStringEnumConverter&lt;TEnum&gt;' instead.</value>
```

Comment:
@eerhardt, is "in Native AOT" the same verbiage we've used elsewhere?

Reply:
I initially had this as "requires dynamic code" but I figured calling out AOT might be more relevant.

Reply:
It's the right verbiage according to https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/. But it is capitalized as `native AOT` I guess?